lib/introspector.js

```diff
@@ -51,7 +51,10 @@ const splitComments = lines => {
 
```

Comment:
It's not directly related to this PR but maybe it's possible to write if-statements in this function simpler.
I've come up with this version. Please tell me your thoughts :) Is it simpler? Is it compatible with syntax (produced example.md is identical)? 
```javascript
const splitComments = lines => {
  let i = 1;
  let hasSignature = false;
  while (i < lines.length && !lines[i].startsWith(' ')) {
    if (lines[i].startsWith('Signature:')) {
      hasSignature = true;
      break;
    }
    i++;
  }
  const signatureStart = i;

  let paramStart = i;
  if (hasSignature) {
    i++;
    while (i < lines.length && lines[i].startsWith(' '.repeat(4))) i++;
    paramStart = i;
  }

  while (i < lines.length && !isNamedLine(lines[i])) i++;
  const commentsStart = i;

  const description = lines.slice(1, signatureStart);
  const argsSignature = lines.slice(signatureStart, paramStart);
  const parameters = lines.slice(paramStart, commentsStart);
  const comments = lines.slice(commentsStart);
  return { description, argsSignature, parameters, comments };
};
```

Reply:
Yeah, it looks much simpler and I don't see issues with compatibility at first sight. But since it is not related to this PR I suggest that you open a new one.